pizzakit-plugin/includes/pizzakit.php

```diff
@@ -1,8 +1,20 @@
 <?php
 
 class Pizzakit {
+	public static $resp = "None";
```

Comment:
Why is this a static class variable?

Reply:
That is a failed attempt to debug the rest api, and should definitly be removed